felix/fv/bpf_test.go

```diff
@@ -3688,6 +3688,9 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 
 					BeforeEach(func() {
 						icmpProto := numorstring.ProtocolFromString("icmp")
+						if testOpts.ipv6 {
```

Comment:
Looks like this code isn't reached due to early return on line 3680?

Reply:
Will be once we enable this test above